tests/testthat/test-renderer2-fill.R

```diff
@@ -0,0 +1,57 @@
+acontext("fill_off")
+
+#
+# test geom with both fill and colour styles
+#
+viz.point <- list(pointone = ggplot() + geom_point(
```

Comment:
can there be more informative names than pointone/two/three please? (something that indicates their purpose, default, fill_off_specified, etc)

Reply:
yes great point! I'll go ahead and rename these.